horace_core/sqw/file_io/@faccess_sqw_v3/put_sqw.m

```diff
@@ -5,74 +5,27 @@
 % (for testing purposes)
 %
 %
-
-
-
-[ok,mess,force_v2,argi]=parse_char_options(varargin,{'-v2'});
+[ok,mess,force_v2,nopix,argi]=parse_char_options(varargin,{'-v2','-nopix'});
 if ~ok
     error('SQW_FILE_IO:invalid_artgument',...
         ['DND_BINFILE_COMMON::put_sqw Error: ',mess]);
 end
 
+if nopix
+    argi{end + 1} = '-nopix';
+end
 obj = put_sqw@sqw_binfile_common(obj,argi{:});
```

Comment:
should put_bytes be called in this function if `~nopix`

Reply:
The idea of `nopix` is that the user of the class can manually call `put_bytes` themselves. I've added a comment that may clarify things